install/fossinit.php

```diff
@@ -166,6 +166,11 @@ function explainUsage()
   require_once("$LIBEXECDIR/resequence_author_table.php"); // If table exists, clean up for Schema
 }
 
+if($isUpdating && substr(str_replace(".", "", $sysconfig['Release']), 0, 3) <= 330)
```

Comment:
-  Why is this better than
  ```php
  if($isUpdating && (empty($sysconfig['Release']) || $sysconfig['Release'] == '3.3.0'))
  ```

- would your code throw a NPE if `$sysconfig['Release'] === null` ?

Reply:
oh, I have overlooked the `<=`.